src/test/java/dailysales/DailySalesTest.java

```diff
@@ -0,0 +1,48 @@
+package dailysales;
+
+import database.FacilityDbGateway;
+import entities.Facility;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.HashMap;
+
+import static org.junit.Assert.assertEquals;
+
+public class DailySalesTest {
+
+    private final DailySalesController dailySalesController = new DailySalesController();
+    private final FacilityDbGateway facilityDbGateway = new FacilityDbGateway();
+    private Facility facility = new Facility("Test Store", "Store");
```

Comment:
The 2nd parameter of the Facility() constructor is not a string, it is of type FacilityType. Please change the 2nd parameter in the constructor to: FacilityType.STORE which should fix the incompatible types error.

Reply:
I fixed this issue in a new branch "DailySalesTest" and created a new pull request